### PR TITLE
Chart mapbox

### PR DESCRIPTION
**Pull request by OwnTooManyMonster**
- Full implementation of Event Listener (Mabox Element change in Chart js)
- deactivated fetch promise all (data access from local storage)
- implementing an automated update routine later

### CODE REVIEW

**Review by soerendolling — APPROVED** [review]
Smooth animations nice. Your links in the navbar show a blue and red decoration. May be this can be removed with decorations:none to the anchor

File: docu.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
-import { cardCreation } from "./src/content/card.js";
+import * as main from "./main.js";
+
+/*import { cardCreation } from "./src/content/card.js";
```
Comment (marwinburesch) [inline docu.js:3]: A comment for your comment: This comment is just commented out code, so I would recommend removing it :)

**Review by marwinburesch — CHANGES_REQUESTED** [review]
I see you're getting started with splitting up your code, excellent. I think you should also take a look at [this review](https://github.com/OwnTooManyMonster/capstone/pull/19#pullrequestreview-660435311), I saw some things again in this PR.

A couple of tips:
- only use `let` when the variable is subject to change later
- The data for your charts could be nicely structured with objects. For example:
```
let NOE00134778date = [];
let NOE00134778value = [];
```

File: base.css [hunk 2]
```diff
@@ -26,26 +23,23 @@
...
   flex-direction: column;
   background-color: transparent;
-
   margin-left: 2%;
 }
 
 .menuIcon {
-  float: right;
-  margin: 3%;
+  margin: 4%;
```
[the remark this answers is not in the document]
Comment (OwnTooManyMonster) [reply base.css:42]: Will try and implement with major css refactoring

File: contact.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import * as main from "./main.js";
```
[the remark this answers is not in the document]
Comment (OwnTooManyMonster) [reply contact.js:1]: Not everything, but everything that is the same on every page!? Used what i know from chess! ;D Open for advise

File: docu.js (hunk 1 above)
Comment (OwnTooManyMonster) [reply docu.js:3]: i know! clean up this week! ;) Thx

**Pull request merged**